Homework11/.idea/compiler.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Too many redundant files in the pull request, please remove them and left only java classes and .pom (maybe you forgot to add .gitignore file to your project?)